File: CSES/introdutory/2-missing-number.c
#include <stdio.h>

int main() 
{
    int n;
    scanf("%d", &n);

    int expected_sum = n * (n + 1) / 2;
    int actual_sum = 0;
    int x;

    for (int i = 0; i < n - 1; i++) {
        scanf("%d", &x);
        actual_sum += x;
    }

    printf("%d\n", expected_sum - actual_sum);

    return 0;
}

File: CSES/2-missing-number.py
n = int(input())
input2 = input()

vet = input2.split(' ')

if (2*(10**5) >= n >= 2) and (len(vet) == n-1):
    for i in range(1, int(len(vet))+2):
        if str(i) not in vet:
            print(i)
            break            


n = int(input())
numbers = list(map(int, input().split()))

expected_sum = n * (n + 1) // 2
actual_sum = sum(numbers)

missing = expected_sum - actual_sum
print(missing)
